pysteps/io/importers.py

```diff
@@ -1432,6 +1452,183 @@ def import_opera_hdf5(filename, qty="RATE", **kwargs):
 
     return precip, quality, metadata
 
+@postprocess_import()
+def import_DPC_hdf5(filename, qty="RATE", **kwargs):
```

Comment:
Should we use this as base for the more general ODIM importer and rename it `import_odim_hdf5` ?